pages/index.js

```diff
@@ -1,30 +1,79 @@
-import React from "react";
+import React, { useState } from 'react';
+import styled from 'styled-components';
 
-import TodoInput from "../components/todo-input";
-import TodoList from "../components/todo-list";
-
-import styled from "styled-components";
+import TodoInput from '../components/todo-input';
+import TodoList from '../components/todo-list';
 
 export default function Home() {
-  return (
-    <Wrapper>
-      <Contents>
-        <TodoInput />
-        <TodoList />
-      </Contents>
-    </Wrapper>
-  );
+	const [date, setDate] = useState();
+	const [todo, setTodo] = useState();
+	const [todos, setTodos] = useState([]);
+	const [id, setID] = useState(0);
+
+	const changeTodo = (e) => {
+		setTodo(e.target.value);
+	};
+
+	const changeDate = (e) => {
+		setDate(e.target.value);
+	};
+
+	const removeTodo = (id) => {
+		const newTodos = [...todos];
+		setTodos(newTodos.filter((info) => info.id !== id));
+	};
```

Comment:
```suggestion
	const handleDelete = (index) => () => {
		setTodos([...todos.slice(0,index), ...todo.slice(index+1, todos.length)]);
	};
```
1. 이벤트 핸들러 이름을 지을 때에는 앞에 접두사 ```handle```을 붙이고 뒤에 기능을 적어주시면 됩니다. [참고](https://blog.sonim1.com/220#%EC%A2%80-%EB%8D%94-%EB%B3%B5%EC%9E%A1%ED%95%9C-%EB%84%A4%EC%9D%B4%EB%B0%8D-more-complicated-naming)
```remove```보다는 crud의 d인 ```delete```로 네이밍 해주시면 좋을 것 같아요.
2. 함수를 위와 같이 정의하고 어떻게 사용하면 되는지는 ```components/todo-list.js```와 ```components/todo-item.js```의 리뷰를 확인해주세요. 
3. ```todos```를 ```map```하면서 매개변수 ```index```에 ```TodoItem```의 index를 넣어줄 예정인데요,
index가 잘 들어왔다고 가정했을 때 ```spread operator```와 ```slice()```를 이용해 업데이트를 깔끔하게 할 수 있습니다. [참고](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)